project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.html

```diff
@@ -163,6 +171,15 @@ <h3 *ngIf="content?.type !== enumContentTypes.ASSESSMENT" class="margin-remove f
         <span class="mat-body-1 margin-left-xxs" *ngIf="content?.complexity"><span>|
             {{ content?.complexity | pipeLimitTo: 300 }}</span>
         </span>
+        <span class="margin-left-xxs margin-top-xxs" *ngIf="content?.progress * 100 === 100; else elseBlock">
```

Comment:
use relative naming for else block template name

Reply:
Fixed  relative names for else block.